integration/spark-common/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala

```diff
@@ -140,6 +142,13 @@ object CarbonReflectionUtils {
         relation,
         expectedOutputAttributes,
         catalogTable)._1.asInstanceOf[LogicalRelation]
+    } else if (SPARK_VERSION.startsWith("2.3")) {
```

Comment:
Add a function SPARK_VERSION.above, so that SPARK_VERSION.above(2.3), so that we need not change code every place for every new version

Reply:
Fixed,added the Utility method for spark version comparison in SparkUtil.scala